modules/040-node-manager/hooks/update_approval.go

```diff
@@ -30,8 +30,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/utils/pointer"
 
-	"github.com/deckhouse/deckhouse/modules/040-node-manager/hooks/internal/shared"
-	ngv1 "github.com/deckhouse/deckhouse/modules/040-node-manager/hooks/internal/v1"
+	"github.com/deckhouse/deckhouse/go_lib/api/shared"
```

Comment:
I assume we can revert all node group changes.

Reply:
Done.